packages/core/test/combinator/continueWith-test.js

```diff
@@ -10,6 +9,6 @@ describe('continueWith', () => {
   it('when f throws, should propagate error', () => {
     const error = new Error()
     const s = continueWith(() => { throw error }, just(0))
-    return drain(s).then(fail, e => assert.same(error, e))
+    return drain(s).then(rejects, eq(error))
```

Comment:
The preferred way to use `rejects` here is:

```js
return rejects(drain(s)).then(is(error))
```

Another way would be to use `@briancavalier/assert`'s `fail`, similar to how it was with buster:

```js
return drain(s).then(fail, is(error))
```

I think I'd also prefer `is` here, although I don't feel too strongly about it.